Buy/Graph.Client.swift

```diff
@@ -59,6 +59,9 @@ extension Graph {
     ///
     public class Client {
         
+        /// Cache policy to use for all `query` operations produced by this instance of `Graph.Client` 
+        public var cachePolicy: CachePolicy = .networkOnly
```

Comment:
would this ever change at runtime? Why not a let and set the default value in the constructor?